go.mod

```diff
@@ -56,12 +56,12 @@ require (
 	github.com/multiformats/go-multihash v0.2.1
 	github.com/onflow/atree v0.4.0
 	github.com/onflow/cadence v0.27.0
-	github.com/onflow/flow v0.3.2
+	github.com/onflow/flow v0.3.3-0.20221012172052-e4ab66320562
 	github.com/onflow/flow-core-contracts/lib/go/contracts v0.11.2-0.20220720151516-797b149ceaaa
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.11.2-0.20220720151516-797b149ceaaa
 	github.com/onflow/flow-go-sdk v0.28.0
 	github.com/onflow/flow-go/crypto v0.24.4
-	github.com/onflow/flow/protobuf/go/flow v0.3.2-0.20220718115540-bd33a4f1cc17
+	github.com/onflow/flow/protobuf/go/flow v0.3.2-0.20221012172052-e4ab66320562
```

Comment:
It would be nice to tag a version to pin here, before merging

Reply:
Can you elaborate on this?
My plan was next:
1. `feature/active-pacemaker` points to specific commit of `onflow/flow`
2. `feature/active-pacemaker` gets merged into master
3. I create a PR in `onflow/flow` to merge branch into `master` and bump version. 
4. Update `go.mod` in `onflow/flow-go` on master branch to point to new version(without targeting specific commit). 

Does that make sense? 
Let me know how do you want to use tag here, currently github.com/onflow/flow/protobuf/go/flow v0.3.2-0.20221012172052-e4ab66320562 targets a commit on specific branch, it's not master.

Reply:
🤦 Of course you're right, shouldn't do this until we're merging the feature branch. Sorry, disregard that suggestion.